src/articles/ExtensionMessage.js

```diff
@@ -43,27 +47,40 @@ export default function ExtensionMessage({
       >
         <Header>
           <Heading>
-            <span className="annotation">New!</span>&nbsp;
-            {strings.extensionHeadline}
+            <span className="annotation">New!</span>&nbsp;Read and translate
```

Comment:
I think we agreed in another discussion that this message with `New!` is going away? It's not "new" anymore anyway :)

Reply:
Done  ✅  But this pop-up will be deleted soon :)